code/implementation/docker/common/src/test/java/io/cattle/platform/docker/client/DockerImageTest.java

```diff
@@ -0,0 +1,31 @@
+package io.cattle.platform.docker.client;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class DockerImageTest {
+    @Test
+    public void testParsingImage(){
```

Comment:
In reality, you are testing imageUuids, which will all start with `docker:`, so you should add versions of all your test cases that start with `docker:` (and `sim:` just to prove backward test compatibility).